docs/glossary.md

```diff
@@ -0,0 +1,184 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description:  Avail common reference for general definitions and concepts.
+
+keywords:
+    - docs
+    - avail
+    - modularity
+    - Monolithic
+    - data availability
+image: https://availproject.github.io/img/avail/AvailDocs.png
+
+
+
+---
+
+# Avail Glossary
+## General
+
+### Modular Blockchain
+A modular blockchain is one that focuses on handling a select few duties and outsources the rest to one or more separate layers.
+
+### Monolithic 
+In a monolithic chain, all tasks are handled on a single layer or a group of tightly coupled chains operating on the same layer. The latter distinction is important to note: a network of interconnected blockchains that handle all roles, such as Polkadot’s parachains or Avalanche’s subnets, doesn’t qualify as modular architecture.
+
+### Data Availability
+Data availability is the guarantee that block proposers publish all transaction data of a block, along with making this data available to all network participants. In essence, it refers to all nodes’ ability to access and download all the data contained within the various blocks of a network.
+
+### Scalability
+Blockchain scalability refers to a blockchain's capacity to execute transactions, store data, and come to consensus as the network's user base grows.
```

Comment:
This is correct in the abstract, but it does not describe Avail very well. Avail does not execute most transactions—rather they are raw data posted to the chain, the execution happens elsewhere (in an execution layer). It's a missed opportunity to talk about precisely what makes Avail special.

I would say that scalability in the Avail context broadly refers to the ability to increase the amount of data published by the chain without negatively impacting the participants and users of the chain. There are two things to call out in particular:
- Ability to grow block size without requiring much more powerful validators and full nodes to process them
- Ability for applications to safely use light clients and to selectively download only data that matters to them, so they aren't affected as block size grows